parachains/runtimes/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -0,0 +1,151 @@
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! The Polkadot Technical Fellowship.
+
+mod origins;
+mod tracks;
+pub use origins::{
+	pallet_origins as pallet_fellowship_origins, Fellows, FellowshipExperts, FellowshipInitiates,
+	FellowshipMasters,
+};
+
+use crate::{
+	constants, impls::ToParentTreasury, weights, AccountId, Balance, Balances, BlockNumber,
+	DotLocation, FellowshipReferenda, Preimage, Runtime, RuntimeCall, RuntimeEvent, Scheduler,
+	DAYS,
+};
+use frame_support::{
+	parameter_types,
+	traits::{EitherOf, MapSuccess, TryMapSuccess},
+};
+use pallet_xcm::{EnsureXcm, IsMajorityOfBody, IsVoiceOfBody};
+use polkadot_runtime_constants::xcm::body::FELLOWSHIP_ADMIN_INDEX;
+use sp_arithmetic::traits::CheckedSub;
+use sp_core::ConstU32;
+use sp_runtime::{
+	morph_types,
+	traits::{AccountIdConversion, ConstU16, Replace, TypedGet},
+};
+use xcm::latest::BodyId;
+
+use self::origins::EnsureFellowship;
+
+/// The Fellowship members' ranks.
+pub mod ranks {
+	use pallet_ranked_collective::Rank;
+
+	pub const INITIATES: Rank = 0;
+	pub const DAN_1: Rank = 1;
+	pub const DAN_2: Rank = 2;
+	pub const DAN_3: Rank = 3; // aka Fellows.
+	pub const DAN_4: Rank = 4;
+	pub const DAN_5: Rank = 5; // aka Experts.
+	pub const DAN_6: Rank = 6;
+	pub const DAN_7: Rank = 7; // aka Masters.
+	pub const DAN_8: Rank = 8;
+	pub const DAN_9: Rank = 9;
+}
+
+parameter_types! {
+	pub const AlarmInterval: BlockNumber = 1;
+	pub const SubmissionDeposit: Balance = 0;
+	pub const UndecidingTimeout: BlockNumber = 7 * DAYS;
+	pub const TechnicalCommittee: BodyId = BodyId::Technical;
+	// Referenda pallet account, used to temporarily deposit slashed imbalance before teleporting.
+	pub ReferendaPalletAccId: AccountId = constants::account::REFERENDA_PALLET_ID.into_account_truncating();
+	pub RelayTreasuryAccId: AccountId = constants::account::RELAY_TREASURY_PALL_ID.into_account_truncating();
+	pub const FellowshipAdminBodyId: BodyId = BodyId::Index(FELLOWSHIP_ADMIN_INDEX);
+}
+
+impl pallet_fellowship_origins::Config for Runtime {}
+
+pub type FellowshipReferendaInstance = pallet_referenda::Instance1;
+
+impl pallet_referenda::Config<FellowshipReferendaInstance> for Runtime {
+	type WeightInfo = weights::pallet_referenda::WeightInfo<Runtime>;
+	type RuntimeCall = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	type Scheduler = Scheduler;
+	type Currency = Balances;
+	type SubmitOrigin =
+		pallet_ranked_collective::EnsureMember<Runtime, FellowshipCollectiveInstance, 1>;
+	type CancelOrigin = FellowshipExperts;
+	type KillOrigin = FellowshipMasters;
+	type Slash = ToParentTreasury<RelayTreasuryAccId, ReferendaPalletAccId, Runtime>;
```

Comment:
I don't think we've ever used `AccId` to mean `AccountId` throughout 6 years in the entire codebase. Don't see why we should start now. Either drop the `Acc` altogether or use the full `Account`.